pagerduty/resource_pagerduty_event_orchestration_path_global_test.go

```diff
@@ -41,6 +41,11 @@ func TestAccPagerDutyEventOrchestrationPathGlobal_Basic(t *testing.T) {
 				Config: testAccCheckPagerDutyEventOrchestrationGlobalDefaultConfig(team, escalationPolicy, service, orch),
 				Check:  resource.ComposeTestCheckFunc(baseChecks...),
 			},
+			{
+				Config: testAccCheckPagerDutyEventOrchestrationGlobalDefaultWithEscalationPolicy(team, escalationPolicy, service, orch),
+				Check:  resource.ComposeTestCheckFunc(baseChecks...),
+			},
```

Comment:
I think I'm understanding how this works, if I added it to the all actions does that suffice, or do you think it's worth adding a specific test for this?

Reply:
Yup, you can just add it to `testAccCheckPagerDutyEventOrchestrationPathGlobalAllActionsConfig`, and `testAccCheckPagerDutyEventOrchestrationPathGlobalAllActionsUpdateConfig`. The reason we have dedicated tests for Automation Actions (`testAccCheckPagerDutyEventOrchestrationPathGlobalAutomationActionsConfig`, `testAccCheckPagerDutyEventOrchestrationPathGlobalAutomationActionsParamsUpdateConfig`, and `testAccCheckPagerDutyEventOrchestrationPathGlobalAutomationActionsParamsDeleteConfig`) is because this action has a more complex config, and we want to verify that adding/updating/deleting nested attributes like parameters and headers works as expected.

Reply:
But I would still add the EP actions to a regular rule in a set and to the catch-all rule, and make one of them reference a Terraformed Escalation Policy: 
```terraform
actions {
...
escalation_policy = pagerduty_escalation_policy.foo.id
...
}
```